sphinxcontrib/inmanta/api.py

```diff
@@ -265,7 +265,7 @@ def emit_resource(self, name: str, cls: Resource, opt: Mapping[str, str]) -> lis
 
         def get_handler(name: str) -> Sequence[type[ResourceHandler]]:
             # ISO8 and pre ISO8 compatiblity
-            handlers = handler.Commander.get_handlers()[name]
+            handlers = handler.Commander.get_handlers().get(name, {})
```

Comment:
Should I raise a warning there?

Reply:
No, it is a valid usecase to not have a handler